python/ray/serve/tests/test_cli.py

```diff
@@ -46,7 +46,15 @@ def assert_deployments_live(names: List[str]):
 
 @pytest.fixture
 def ray_start_stop():
+    subprocess.check_output(["ray", "stop", "--force"])
     subprocess.check_output(["ray", "start", "--head"])
+    wait_for_condition(
```

Comment:
Could we ping the `/api/ray/version` endpoint here instead of the `/api/serve/deployments/` endpoint? See this comment for context.